src/tmt_web/api.py

```diff
@@ -1,29 +1,107 @@
+import json
+import logging
 import os
-from typing import Annotated, Any, Literal
+import platform
+import time
+from datetime import UTC, datetime
+from typing import Annotated, Literal
 
 from celery.result import AsyncResult
-from fastapi import FastAPI
+from fastapi import FastAPI, HTTPException, Request, status
 from fastapi.params import Query
+from fastapi.responses import HTMLResponse, JSONResponse, PlainTextResponse, RedirectResponse
 from pydantic import BaseModel
-from starlette.responses import HTMLResponse, JSONResponse, PlainTextResponse
+from tmt import Logger
+from tmt import __version__ as tmt_version
+from tmt.utils import GeneralError, dict_to_yaml
 
 from tmt_web import service, settings
 from tmt_web.generators import html_generator
 
-app = FastAPI()
+# Record start time for uptime calculation
+START_TIME = time.time()
+
+# Create main logger for the API
+logger = Logger(logging.getLogger("tmt-web-api"))
+
+app = FastAPI(
+    title="tmt Web API",
+    description="Web API for checking tmt tests, plans and stories",
```

Comment:
It would be fine to add a link to tmt-web repository + add info about where to report service issues.

Reply:
Not sure about service issues, but is this ok with you?
![image](https://github.com/user-attachments/assets/66b0706d-d882-4a16-9e63-53f5a575b67f)